### ISSUE TITLE
Nested Modals will break scrolling

### ISSUE THREAD

**Issue by obiwan007** [dougludlow/ng2-bs3-modal#57]
If you have a really high/large modal and open a modal on top if it, the scrolling is broken if you close the top form again.
Maybe some infos on how to fix:
http://stackoverflow.com/questions/19305821/multiple-modals-overlay

**Comment by dougludlow** [comment]
@obiwan007 thanks for finding this. I'm a little swamped on other projects so I probably won't be able to take a look at this for a couple weeks. Just FYI.

**Comment by mounikaymrs** [comment]
I have the same issue. I came across several hacky solutions but not a proper one yet.

**Comment by fxcosta** [comment]
I do not know if this is the best way to solve it, but in my case it worked something similar to this:

```
        document.querySelector('body').classList.add('modal-open');
```

On onClose and onDismiss events in a component method.

**Comment by dougludlow** [comment]
This should finally be fixed in v0.12. Enjoy.
